### PR TITLE
Companion to "hook in transition location properly".

### PR DESCRIPTION
**Pull request by vext01**
This is the companion to https://github.com/ykjit/yk/pull/454.

It's just for demonstration purposes at this point, so don't do a detailed review.

### CODE REVIEW

File: llvm/lib/Transforms/Yk/ControlPoint.cpp [hunk 1]
```diff
@@ -1,5 +1,7 @@
 //===- ControlPoint.cpp - Synthesise the yk control point -----------------===//
 //
+// FIXME: update this comment.
```
Comment (ltratt) [inline llvm/lib/Transforms/Yk/ControlPoint.cpp:3]: Is there a reason we're not updating it in this PR?

Comment (vext01) [reply llvm/lib/Transforms/Yk/ControlPoint.cpp:3]: Only that I've been focusing on the runtime side.

I'll need to get this one up to scratch before we can merge https://github.com/ykjit/ykllvm/pull/17

Comment (ltratt) [reply llvm/lib/Transforms/Yk/ControlPoint.cpp:3]: This is #17?

Comment (vext01) [reply llvm/lib/Transforms/Yk/ControlPoint.cpp:3]: sorry, i meant, before we can merge https://github.com/ykjit/yk/pull/454

Comment (ltratt) [reply llvm/lib/Transforms/Yk/ControlPoint.cpp:3]: I'm also unclear: are you going to fix the comment or ... ?

Comment (vext01) [reply llvm/lib/Transforms/Yk/ControlPoint.cpp:3]: Yes, I'll fix it tomorrow morning :)

**Comment by vext01** [comment]
2ec45596 updates the large comment at the top of the file.

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
Squashed and wrote a proper commit message.

**Pull request merged**